package.json

```diff
@@ -4,8 +4,7 @@
   "main": "./index.js",
   "typings": "./index.d.ts",
   "scripts": {
-    "precontrib": "npm install -g typings typescript && npm install",
-    "contrib": "typings install && tsc"
+    "prepublish": "typings install && tsc"
```

Comment:
calling tsc after installing it via devDependencies does not work for me.
am I missing something here? maybe it worked for you because you have typescript installed globally?